spec.md

```diff
@@ -209,7 +209,7 @@ If the blob or manifest is not found in the registry, the response code MUST be
 Pushing an object typically works in the opposite order as a pull: the blobs making up the object are uploaded first, and the manifest last.
 A useful diagram is provided [here](https://github.com/google/go-containerregistry/tree/d7f8d06c87ed209507dd5f2d723267fe35b38a9f/pkg/v1/remote#anatomy-of-an-image-upload).
 
-A registry MUST accept an otherwise valid manifest with a `subject` field that references a manifest that does not exist, allowing clients to push a manifest and referrers to that manifest in either order.
+A registry MUST initially accept an otherwise valid manifest with a `subject` field that references a manifest that does not exist in the repository, allowing clients to push a manifest and referrers to that manifest in either order.
```

Comment:
Also add the language?

The `subject` relationship is a weak association, pls see image spec, etc?

Reply:
@rchincha how should that be phrased here?

Reply:
@rchincha would you be opposed to leaving that off for a subsequent patch or minor version, as we look to clarify more language over time?

I think it'd be fine to add it, depending on wording, but it might stand out and lead to more questions when comparing the two specs. Just devil's advocate, I think it'd be fine either way.

Reply:
weak / strong are certainly common vernacular but would raise more questions.. the also probably not the right place to define subject descriptor reference..

Reply:
see above subject definition:
`- **Subject**: an association from one manifest to another, typically used to attach an artifact to an image.`

maybe in another PR and change ^ to `is an association, aka. a weak reference, from one manifest to another, typically used to attach an artifact to an image.`

but that implies we should use strong for content descriptors... yada yada

Reply:
but then we'd have to explain the these weak references may still be verified as a part of a transaction ...

Reply:
given the new definition in image.. I think we can just link to that definition and/or normalize in distribution's definition by saying a weak association in the definition we have here...